Python Package set updates

mweinelt opened the pull request:
Regular python-unstable run with major updates to libraries.

https://hydra.nixos.org/jobset/nixpkgs/python-unstable

- [x] Squashed fixup commits before merge

andersk committed:
python3Packages.isort: 5.6.4 → 5.9.3

Signed-off-by: Anders Kaseorg <andersk@mit.edu>

fabaff committed:
python3Packages.starlette: 0.14.2 -> 0.16.0

fabaff committed:
python3Packages.asn1: init at 2.4.1

fabaff committed:
python3Packages.smart-meter-texas: 0.4.4 -> 0.4.6

austinbutler committed:
python3Packages.poetry-core: 1.0.3 -> 1.0.4

austinbutler committed:
python3Packages.poetry: 1.1.7 -> 1.1.8

mweinelt committed:
python3Packages.simple-salesforce: 1.11.2 -> 1.11.3

mweinelt committed:
python3Packages.simplejson: 3.17.4 -> 3.17.5

mweinelt committed:
python3Packages.singledispatch: 3.6.2 -> 3.7.0

mweinelt committed:
python3Packages.screeninfo: 0.6.7 -> 0.7

mweinelt committed:
python3Packages.setuptools_scm: add regression note

Updating to 6.1.0 and 6.2.0 causes regressions.

mweinelt committed:
python3Packages.tempora: 4.0.2 -> 4.1.1

mweinelt committed:
python3Packages.ssdp: 1.0.1 -> 1.1.0

mweinelt committed:
python3Packages.terminado: 0.10.1 -> 0.11.1

mweinelt committed:
python3Packages.tableaudocumentapi: 0.6 -> 0.7

mweinelt committed:
python3Packages.tensorboard_plugin_profile: 2.4.0 -> 2.5.0

mweinelt committed:
python3Packages.sqlite-utils: 3.9.1 -> 3.17

mweinelt committed:
python3Packages.sphinxcontrib-tikz: 0.4.13 -> 0.4.14

mweinelt committed:
python3Packages.slack-sdk: 3.10.0 -> 3.10.1

mweinelt committed:
python3Packages.smart-meter-texas: 0.4.4 -> 0.4.7

mweinelt committed:
python3Packages.sphinx-autoapi: 1.8.1 -> 1.8.4

mweinelt committed:
python3Packages.testtools: 2.4.0 -> 2.5.0

mweinelt committed:
python3Packages.tox: 3.23.1 -> 3.24.3

mweinelt committed:
python3Packages.texttable: 1.6.3 -> 1.6.4

mweinelt committed:
python3Packages.tqdm: 4.61.1 -> 4.62.2

mweinelt committed:
python3Packages.trytond: 5.8.9 -> 5.8.11

mweinelt committed:
python3Packages.teslajsonpy: 0.18.3 -> 0.19.0

mweinelt committed:
python3Packages.types-pytz: 2021.1.0 -> 2021.1.2

mweinelt committed:
python3Packages.tern: 2.6.1 -> 2.7.0

mweinelt committed:
python3Packages.termplotlib: 0.3.5 -> 0.3.8

mweinelt committed:
python3Packages.tifffile: 2021.6.14 -> 2021.8.8

mweinelt committed:
python3Packages.twine: 3.4.1 -> 3.4.2

mweinelt committed:
python3Packages.xarray: 0.18.2 -> 0.19.0

mweinelt committed:
python3Packages.watchdog: 2.1.4 -> 2.1.5

mweinelt committed:
python3Packages.uvloop: 0.15.2 -> 0.16.0

mweinelt committed:
python3Packages.webargs: 8.0.0 -> 8.0.1

mweinelt committed:
python3Packages.vispy: 0.6.6 -> 0.8.0

mweinelt committed:
python3Packages.yamlloader: 1.0.0 -> 1.1.0

mweinelt committed:
python3Packages.xml2rfc: 3.8.0 -> 3.9.1

mweinelt committed:
python3Packages.vowpalwabbit: 8.10.2 -> 8.11.0

mweinelt committed:
python3Packages.ytmusicapi: 0.18.0 -> 0.19.1

mweinelt committed:
python3Packages.virtualenv: 20.4.7 -> 20.7.2

mweinelt committed:
python3Packages.zope.proxy: 4.3.5 -> 4.4.0

mweinelt committed:
python3Packages.zipp: 3.4.1 -> 3.5.0

mweinelt committed:
python3Packages.zope.component: 5.0.0 -> 5.0.1

mweinelt committed:
python3Packages.zarr: 2.9.1 -> 2.9.3

mweinelt committed:
python3Packages.zigpy-znp: 0.5.3 -> 0.5.4

mweinelt committed:
python3Packages.aioprocessing: 1.1.0 -> 2.0.0

mweinelt committed:
python3Packages.aioredis: 1.3.1 -> 2.0.0

mweinelt committed:
python3Packages.azure-mgmt-recoveryservices: 1.0.0 -> 2.0.0

mweinelt committed:
python3Packages.azure-mgmt-iothubprovisioningservices: 0.3.0 -> 1.0.0

mweinelt committed:
python3Packages.azure-mgmt-apimanagement: 0.3.0 -> 2.1.0

mweinelt committed:
python3Packages.ansible-runner: 1.4.7 -> 2.0.2

mweinelt committed:
python3Packages.apispec: 4.7.1 -> 5.1.0

mweinelt committed:
python3Packages.casbin: 0.20.0 -> 1.5.0

mweinelt committed:
python3Packages.django-cacheops: 5.1 -> 6.0

mweinelt committed:
python3Packages.cmd2: 1.5.0 -> 2.1.2

mweinelt committed:
python3Packages.chart-studio: 4.14.3 -> 5.2.2

mweinelt committed:
python3Packages.decorator: 4.4.2 -> 5.0.9

mweinelt committed:
python3Packages.contextlib2: 0.6.0.post1 -> 21.6.0

mweinelt committed:
python3Packages.colorlog: 5.0.1 -> 6.4.1

mweinelt committed:
python3Packages.cupy: 8.6.0 -> 9.4.0

mweinelt committed:
python3Packages.block-io: 1.1.15 -> 2.0.4

mweinelt committed:
python3Packages.ipykernel: 5.5.5 -> 6.2.0

mweinelt committed:
python3Packages.jira: 2.0.0 -> 3.0.1

mweinelt committed:
python3Packages.frozendict: 1.2 -> 2.0.5

mweinelt committed:
python3Packages.islpy: 2020.2.2 -> 2021.1

mweinelt committed:
python3Packages.IMDbPY: 2020.9.25 -> 2021.4.18

mweinelt committed:
python3Packages.gcovr: 4.2 -> 5.0

mweinelt committed:
python3Packages.kubernetes: 12.0.1 -> 18.20.0

mweinelt committed:
python3Packages.labelbox: 2.7.0 -> 3.1.0

mweinelt committed:
python3Packages.jupyter_client: 6.2.0 -> 7.0.1

mweinelt committed:
python3Packages.nbdime: 2.1.0 -> 3.1.0

mweinelt committed:
python3Packages.mitmproxy: 6.0.2 -> 7.0.2

mweinelt committed:
python3Packages.marshmallow-oneofschema: 2.1.0 -> 3.0.1

mweinelt committed:
python3Packages.plotly: 4.14.3 -> 5.2.2

mweinelt committed:
python3Packages.lexid: 2020.1005 -> 2021.1006

mweinelt committed:
python3Packages.pluggy: 0.13.1 -> 1.0.0

mweinelt committed:
python3Packages.plaid-python: 7.5.0 -> 8.0.0

mweinelt committed:
python3Packages.libarchive-c: 2.9 -> 3.1

mweinelt committed:
python3Packages.sphinxcontrib-htmlhelp: 1.0.3 -> 2.0.0

mweinelt committed:
python3Packages.rdflib: 5.0.0 -> 6.0.0

mweinelt committed:
python3Packages.pytest-isort: 1.3.0 -> 2.0.0

mweinelt committed:
python3Packages.pytube: 10.9.3 -> 11.0.0

mweinelt committed:
python3Packages.pudb: 2020.1 -> 2021.1

mweinelt committed:
python3Packages.spyder: 4.2.5 -> 5.1.1

mweinelt committed:
python3Packages.Pykka: 2.0.3 -> 3.0.1

mweinelt committed:
python3Packages.pyramid: 1.10.8 -> 2.0

mweinelt committed:
python3Packages.priority: 1.3.0 -> 2.0.0

mweinelt committed:
python3Packages.trytond: 5.8.11 -> 6.0.5

mweinelt committed:
python3Packages.spyder-kernels: 1.10.3 -> 2.1.0

mweinelt committed:
python3Packages.yt: 3.6.1 -> 4.0.1

mweinelt committed:
python3Packages.vivisect: 0.2.1 -> 1.0.4

mweinelt committed:
python3Packages.towncrier: 19.9.0 -> 21.3.0

mweinelt committed:
python3Packages.sure: 1.4.11 -> 2.0.0

mweinelt committed:
python3Packages.pytest: relax pluggy constraints

Apparently possible without further changes:
https://github.com/pytest-dev/pytest/pull/9040

mweinelt committed:
python2Packages.contextlib2: reinit at 0.6.0.post1

mweinelt committed:
python2Packages.pluggy: reinit at 0.13.1

SuperSandro2000 committed:
python39Packages.backports-entry-points-selectable: init at 1.1.0

SuperSandro2000 committed:
python3Packages.pytest_4: relax pluggy constraints

Following 7e27631e46c8121e5b906083e1de74ec53de2730

mweinelt committed:
python3Packages.brotli: convert to pytestCheckHook

mweinelt committed:
python3Packages.requests: restore python2 compatibility

SuperSandro2000 committed:
python3Packages: provide setuptools-scm with toml[i] by default and remove duplicated toml packages

Some packages seem to still require toml despite setuptools-scm having switched to tomli.
If it is missing the version number in dist.into is set to 0.0.0 and silently all version pins break.

SuperSandro2000 committed:
python39Packages.zipp: update meta, remove unused checkInputs

tests are not run anyway

SuperSandro2000 committed:
python39Packages.setuptools-scm: relax tomli ping

SuperSandro2000 committed:
python39Packages.tomli: break infinite recursion with setuptools-scm by splitting tests, add SuperSandro2000 as maintainer

SuperSandro2000 committed:
python39Packages.flit-core: break infinite recursion with setuptools-scm by splitting tests, add SuperSandro2000 as maintainer

mweinelt committed:
python3Packages.pytest_5: relax pluggy constraints

SuperSandro2000 committed:
python39Packages.Cython: 0.29.22 -> 0.29.24

Required to build numpy

RuntimeError: Building NumPy requires Cython >= 0.29.24, found 0.29.22 at /nix/store/7qrl7hhzvbmd966b61w3n0dkamn863gn-python3.9-Cython-0.29.22/lib/python3.9/site-packages/Cython/init.py

SuperSandro2000 committed:
python39Packages.Cython: minor formatting cleanups

mweinelt committed:
python3Packages.sphinx: fix build with Pygments>=2.10

SuperSandro2000 committed:
python3Packages.pytest_6_1: relax pluggy constraints

SuperSandro2000 committed:
python3Packages.async-upnp-client: use latest pytest again

SuperSandro2000 committed:
python3Packages.trio: use latest pytest again

SuperSandro2000 committed:
python3Packages.urrllib3: use latest pytest again

SuperSandro2000 committed:
python3Packages.werkzeug: use latest pytest again

SuperSandro2000 committed:
python39Packages.fakeredis: disable failing tests

SuperSandro2000 committed:
python39Packages.gevent-websocket: update meta, disable checkPhase, add import check

SuperSandro2000 committed:
python39Packages.zigpy: 0.36.1 -> 0.37.1

SuperSandro2000 committed:
python39Packages.httpx: 0.18.2 -> 0.19.0

SuperSandro2000 committed:
python39Packages.Fabric: format, use pytestCheckHook, update disable test comment

SuperSandro2000 committed:
python39Packages.paramiko: use pytestCheckHook, cleanup

SuperSandro2000 committed:
python39Packages.pytest-relaxed: switch to pytestCheckHook, relax decorator dependency, comment partial breakage

SuperSandro2000 committed:
python39Packages.tzlocal: add a comment to be compatible with APScheduler

SuperSandro2000 committed:
python39Packages.zigpy-xbee: 0.13.0 -> 0.14.0

SuperSandro2000 committed:
python3Packages.wasmer*: package entire wasmer-compiler family

SuperSandro2000 committed:
python39Packages.fastapi: format, disable failing test

SuperSandro2000 committed:
python39Packages.keyring: disable failing test

SuperSandro2000 committed:
python39Packages.plotly: update meta, add tenacity hard depedency, removed no longer mentioned optional depedencies

SuperSandro2000 committed:
wapiti: disable failing test

SuperSandro2000 committed:
python39Packages.sanic-testing: loose httpx constraint

SuperSandro2000 committed:
python39Packages.slowapi: disable failing tests

mweinelt committed:
python3Packages.botocore: 1.20.97 -> 1.20.106

mweinelt committed:
python3Packages.boto3: 1.17.97 -> 1.17.106

mweinelt committed:
awscli: 1.19.97 -> 1.19.106

SuperSandro2000 committed:
platformio: pin ajsonrpc dependency

SuperSandro2000 committed:
maturin: 0.10.6 -> 0.11.3

mweinelt committed:
python3Packages.pylitterbot: 2021.8.0 -> 2021.8.1

SuperSandro2000 committed:
python39Packages.bellows: disable failing tests

mweinelt committed:
python2Packages.decorator: reinit at 4.4.2

mweinelt committed:
home-assistant: relax httpx, requests constraints

jonringer committed:
python3Packages.azure-mgmt-iothubprovisioningservices: fix build

jonringer committed:
azure-cli: fix build

mweinelt committed:
python3Packages.immutabledict: 2.1.0 -> 2.2.0

mweinelt committed:
python3Packages.pykka: fix build, tests and refactor

Tests were not available in the PyPi package anymore.

mweinelt committed:
python3Packages.ipykernel: add missing deps and split off tests

The package now requires debugpy and the tests require ipyparallel.

The latter causes an infinite recursion, which is why I split out the
tests into `passthru.tests.pytest`. There is not dedicated tests output,
because the tests require relative imports.

SuperSandro2000 commented on pkgs/tools/admin/azure-cli/python-packages.nix:
```diff
@@ -98,7 +99,7 @@ let
             --ignore=azure/cli/core/tests/test_profile.py \
             --ignore=azure/cli/core/tests/test_generic_update.py \
-            -k 'not metadata_url'
+            -k 'not metadata_url and not test_send_raw_requests'
```
This could be converted to pytestCheckHook.

SuperSandro2000 commented on pkgs/development/python-modules/pyphen/default.nix:
```diff
@@ -1,14 +1,26 @@
   };
 
+  preCheck = ''
+    sed -i '/addopts/d' pyproject.toml
```
Why do we need this? I didn't see such change before.

mweinelt replied:
It deletes pytest arguments.

https://github.com/Kozea/Pyphen/blob/master/pyproject.toml#L48

SuperSandro2000 commented on pkgs/development/python-modules/ipykernel/tests.nix:
```diff
@@ -0,0 +1,57 @@
+    flaky
+    ipykernel
+    ipyparallel
+    nose
+    pytestCheckHook
```
You are sure the tests get upstream not executed by nose?

mweinelt replied:
I don't understand what you are saying.

Does the test suite need both pytest and nose? Yes.

SuperSandro2000 committed:
python39Packages.fastdiff: mark broken on 32bit

SuperSandro2000 committed:
platformio: remove 32bit packages to unbreak

SuperSandro2000 committed:
python39Packages.xdis: 5.0.10 -> 5.0.11

SuperSandro2000 committed:
python39Packages.debugpy: throw on unsupported system

this fixes failures for example in nixpkgs-review on aarch64-linux

SuperSandro2000 committed:
python39Packages.sphinxcontrib-tikz: replace patch by substituteInPlace to prevent apply errors

SuperSandro2000 committed:
python39Packages.zodb: remove outdated disabled

SuperSandro2000 committed:
vulnix: disable python3Packages overwrite

SuperSandro2000 committed:
python39Packages.zconfig: 3.5.0 -> 3.6.0

SuperSandro2000 committed:
top-level: use python3Packages instead of python37Packages

SuperSandro2000 committed:
variety: use python3Packages instead of python37Packages

SuperSandro2000 committed:
boxfort: use python3Packages instead of python37Packages

SuperSandro2000 committed:
criterion: use python3Packages instead of python37Packages

SuperSandro2000 committed:
appgate-sdp: use python3Packages instead of python37Packages

mweinelt committed:
plover.dev: 4.0.0.dev8 -> 4.0.0.dev10

Also unpins the aging python36Packages.

risicle commented:
Hmm. On macos 10.15:
```
builder for '/nix/store/rmv0vw9p7ynb7avnsf1yjr57701ha68i-python3.8-pythran-0.9.12.drv' failed with exit code 1
...
OSError: dlopen(/nix/store/24pj2503xx3hspq0jxzfs315p545w8a1-clang-7.1.0-lib/lib/libgomp.dylib, 6): image not found
```

mweinelt committed:
python3Packages.mocket: relax decorator constraint

Disables a failing test case, because of the relaxed constraint. The
geoip2 package, the only consumer of this package, does not exhibit any
problems resulting from this change.

mweinelt committed:
python3Packages.geoalchemy2: fix build, enable tests, refactor

mweinelt committed:
python3Packages.pytest-doctestplus: disable failing tests, update deps

risicle committed:
python3Packages.pyosmium: add requests dependency

mweinelt committed:
python3Packages.commoncode: 21.7.23 -> 21.8.27

mweinelt committed:
python3Packages.alembic: 1.6.5 -> 1.7.1

mweinelt committed:
python3Packages.casbin: 1.5.0 -> 1.7.0

mweinelt committed:
python3Packages.cfn-lint: 0.53.0 -> 0.53.1

mweinelt committed:
python3Packages.django-environ: 0.4.5 -> 0.5.0

mweinelt committed:
python3Packages.chart-studio: 5.2.2 -> 5.3.0

mweinelt committed:
python3Packages.astroid: 2.7.2 -> 2.7.3

mweinelt committed:
python3Packages.google-cloud-bigquery-datatransfer: 3.3.1 -> 3.3.2

mweinelt committed:
python3Packages.google-cloud-translate: 3.3.2 -> 3.4.0

mweinelt committed:
python3Packages.google-cloud-firestore: 2.3.0 -> 2.3.1

mweinelt committed:
python3Packages.google-cloud-speech: 2.7.0 -> 2.8.0

mweinelt committed:
python3Packages.google-cloud-monitoring: 2.4.2 -> 2.5.0

mweinelt committed:
python3Packages.flexmock: 0.10.7 -> 0.10.8

mweinelt committed:
python3Packages.google-cloud-spanner: 3.8.0 -> 3.9.0

mweinelt committed:
python3Packages.google-cloud-kms: 2.5.0 -> 2.6.0

mweinelt committed:
python3Packages.google-cloud-os-config: 1.3.2 -> 1.4.0

mweinelt committed:
python3Packages.google-cloud-testutils: 1.0.0 -> 1.1.0

mweinelt committed:
python3Packages.dpkt: 1.9.7.1 -> 1.9.7.2

mweinelt committed:
python3Packages.minidump: 0.0.18 -> 0.0.19

mweinelt committed:
python3Packages.google-crc32c: 1.1.2 -> 1.1.3

mweinelt committed:
python3Packages.kiwisolver: 1.3.1 -> 1.3.2

mweinelt committed:
python3Packages.ipykernel: 6.2.0 -> 6.3.0

mweinelt committed:
python3Packages.labelbox: 3.1.0 -> 3.2.0

mweinelt committed:
python3Packages.mautrix: 0.10.5 -> 0.10.6

mweinelt committed:
python3Packages.ipython: 7.26.0 -> 7.27.0

mweinelt committed:
python3Packages.gradient: 1.7.2 -> 1.7.4

mweinelt committed:
python3Packages.jupyter_client: 7.0.1 -> 7.0.2

mweinelt committed:
python3Packages.hypothesis: 6.15.0 -> 6.17.3

mweinelt committed:
python3Packages.py3status: 3.38 -> 3.39

mweinelt committed:
python3Packages.mne-python: 0.23.2 -> 0.23.3

mweinelt committed:
python3Packages.plotly: 5.2.2 -> 5.3.0

mweinelt committed:
python3Packages.plaid-python: 8.0.0 -> 8.1.0

mweinelt committed:
python3Packages.pyfakefs: 4.5.0 -> 4.5.1

mweinelt committed:
python3Packages.pytest: 6.2.4 -> 6.2.5

mweinelt committed:
python3Packages.prettytable: 2.1.0 -> 2.2.0

mweinelt committed:
python3Packages.python-gitlab: 2.10.0 -> 2.10.1

mweinelt committed:
python3Packages.pyfaidx: 0.6.1 -> 0.6.2

mweinelt committed:
python3Packages.qcelemental: 0.21.0 -> 0.22.0

mweinelt committed:
python3Packages.rich: 10.7.0 -> 10.9.0

mweinelt committed:
python3Packages.regex: 2021.8.21 -> 2021.8.28

mweinelt committed:
python3Packages.pytube: 11.0.0 -> 11.0.1

mweinelt committed:
python3Packages.traitlets: 5.0.5 -> 5.1.0

mweinelt committed:
python3Packages.smart-open: 5.2.0 -> 5.2.1

mweinelt committed:
python3Packages.vispy: 0.8.0 -> 0.8.1

mweinelt committed:
python3Packages.sagemaker: 2.56.0 -> 2.57.0

mweinelt committed:
python3Packages.typing_extensions: 3.10.0.0 -> 3.10.0.2

mweinelt committed:
python3Packages.sparse: 0.12.0 -> 0.13.0

mweinelt committed:
python3Packages.ROPGadget: 6.5 -> 6.6

mweinelt committed:
python3Packages.snowflake-connector-python: 2.5.1 -> 2.6.0

mweinelt committed:
python3Packages.ruamel.yaml: 0.17.14 -> 0.17.16

mweinelt committed:
python3Packages.zarr: 2.9.3 -> 2.9.4

mweinelt committed:
python3Packages.pydeconz: 82 -> 83

mweinelt committed:
python3Packages.pyTelegramBotAPI: 3.8.3 -> 4.0.0

mweinelt committed:
python3Packages.freetype-py: enable tests

mweinelt committed:
python3Packages.mocket: 3.9.42 -> 3.9.44

SuperSandro2000 committed:
kupfer: no longer use python36Packages

SuperSandro2000 committed:
python39Packages.canonicaljson: execute tests with pytestCheckHook, disable failing test

SuperSandro2000 committed:
python39Packages.ibm-cloud-sdk-core: disable failing tests, remove linting and not required checkInputs

SuperSandro2000 committed:
python39Packages.dateutils: init at 0.6.12

SuperSandro2000 committed:
python39Packages.rdflib-jsonld: mark broken

SuperSandro2000 committed:
python39Packages.python-lsp-server: disable failing test after pylint update, format, move postPatch after src

SuperSandro2000 committed:
python39Packages.pyocr: switch to pytestCheckHook, format, fix setuptools not finding version

SuperSandro2000 committed:
pylode: 2.8.6 -> 2.12.0

SuperSandro2000 committed:
python39Packages.moviepy: relax decorator dependency, forma, remove ? null from inputs

SuperSandro2000 committed:
python39Packages.fiona: fix tests, format, normalise pname

SuperSandro2000 committed:
python39Packages.werkzeug: normalise pname

SuperSandro2000 committed:
caffe2: mark broken

SuperSandro2000 committed:
python39Packages.fastparquet: 0.7.0 -> 0.7.1

fabaff committed:
python3Packages.elmax: 0.1.2 -> 0.1.3

fabaff committed:
python3Packages.hstspreload: 2021.8.1 -> 2021.9.1

fabaff committed:
python3Packages.platformdirs: 2.2.0 -> 2.3.0

fabaff committed:
python3Packages.ssdp: fix build

fabaff committed:
bump2version: 1.0.0 -> 1.0.1

fabaff committed:
python3Packages.cmd2: fix build

mweinelt committed:
python3Packages.dask: 2021.06.2 -> 2021.08.1

fabaff committed:
python3Packages.acme-tiny: fix build

fabaff committed:
python3Packages.acoustics: fix build

fabaff committed:
python3Packages.asttokens: 2.0.4 -> 2.0.5

fabaff committed:
python3Packages.k5test: fix build

mweinelt committed:
python3Packages.executing: fix build

Adds missing setuptools-scm dependency.

fabaff committed:
python3Packages.screeninfo: fix build

mweinelt commented on pkgs/development/python-modules/hdbscan/default.nix:
```diff
@@ -44,6 +44,8 @@ buildPythonPackage rec {
     "test_hdbscan_boruvka_balltree_matches"
   ];
 
+  pythonImportsCheck = [ "hdbscan" ];
```
Sneaky!

fabaff replied:
Ups, that ended accidentally in that commit.

mweinelt replied:
Done.

fabaff committed:
python3Packages.quantities: fix build

fabaff committed:
snscrape: 0.3.4 -> unstable-2021-08-30

mweinelt committed:
python3Packages.python-daemon: fix build & refactor

mweinelt committed:
calibre-web: relax requests constraints

mweinelt committed:
python3Packages.hdbscan: add imports check
